CHANGELOG.md

```diff
@@ -2,23 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased - as of Sprint 23 end 2015-05-11
+## Unreleased
```

Comment:
Can you make this a `#` instead of `##` to match the Botvinnik line?

Also, I like this layout much better.